misc/requirements/requirements-pyinstaller.txt

```diff
@@ -1,5 +1,5 @@
 # This file is automatically generated by scripts/dev/recompile_requirements.py
 
 altgraph==0.17
-pyinstaller==4.3
+pyinstaller==4.4
```

Comment:
unrelated change

Reply:
Yep! Those get [updated automatically each week](https://github.com/qutebrowser/qutebrowser/pulls?q=is%3Apr+author%3Aqutebrowser-bot), so usually `recompile_requirements.py` only needs to be run manually when adding a new dependency (or removing on).